src/pages/Introduce/Introduce.scss

```
@@ -0,0 +1,51 @@
+@import url("../../style/common.scss");
+@import url("../../style/reset.scss");
+
+@mixin center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.introduce {
+  @include center;
+  width: 250px;
+  height: 100vh;
+  margin: 0 auto;
+  font-size: 14px;
+  .introduce-box {
+    flex-direction: column;
+    padding: 14px;
+    border: 1px solid black;
+    border-radius: 5px;
+  }
+  .introduce-wecode {
```

```suggestion
  font-size: 14px;
  
  .introduce-box {
    flex-direction: column;
    padding: 14px;
    border: 1px solid black;
    border-radius: 5px;
  }
  
  .introduce-wecode {
```

가독성을 위해서 스타일끼리 한칸씩 띄워주세요!